docs/sources/mimir/references/http-api/index.md

```diff
@@ -566,8 +566,17 @@ GET,POST <prometheus-http-prefix>/api/v1/cardinality/label_values
 Returns realtime label values cardinality associated to request param `label_names[]` across all ingesters, for the authenticated tenant, in `JSON` format.
 It returns the series count per label value associated to request param `label_names[]`.
 
-As far as this endpoint generates cardinality report using only values from currently opened TSDBs in ingesters, two subsequent calls may return completely different results, if ingester did a block
-cutting between the calls.
+Two methods of counting are available, `inmemory` and `active`. This choice is controlled via the `count_method` parameter.
+
+The `inmemory` counting method counts the number of series using all values from currently opened TSDBs in ingesters.
+Two subsequent calls may return completely different results, if ingester did a block cutting between the calls.
+This method of counting is most useful for understanding ingester memory usage.
+
+The `active` counting method counts also uses values from currently opened TSDBs in ingesters, but it filters out series that have not received a sample within active series idle timeout.
+The duration of this timeout is controlled via the `ingester.active-series-metrics-idle-timeout` parameter.
+This method of counting is most useful for understanding which label values are contributing to the current sample ingestion in Mimir.
```

Comment:
```suggestion
This method of counting is most useful for understanding which label values are represented in the samples ingested by Mimir in the last `ingester.active-series-metrics-idle-timeout` minutes.
```

I'm not necessarily sure that its minutes but does this phrasing work?

Reply:
Yep, the unit is specified in the value. So you can say `5m` to mean 5 minutes or `30s` to mean 30 seconds.